frame/staking/src/benchmarking.rs

```diff
@@ -502,7 +502,7 @@ benchmarks! {
 			compact,
 			score,
 			size
-		) = offchain_election::prepare_submission::<T>(assignments, winners, false).unwrap();
+		) = offchain_election::prepare_submission::<T>(assignments, winners, false, T::MaximumBlockWeight::get()).unwrap();
```

Comment:
Should rather be `MaximumExtrinsicWeight`?

Reply:
maybe, although in the context of benchmarks this is irrelevant, so I would just leave it as is.

Reply:
It probably affects the calculated weight cost, but I guess it's better to be on the safe side. That said, with operational the transaction can actually be greater than `MaximumBlockWeight` now.